### PR TITLE
Create build.yml

### LINKED ISSUE #57: Set up build action on pull request
Set up a build action for this repository to ensure that the build passes before CD is triggered

### PR DESCRIPTION
**Pull request by JamesCahyadi**
Created a build.yml file which is based off the standard template workflow as mentioned here: https://docs.github.com/en/actions/guides/building-and-testing-nodejs

Basically it will run npm ci (which is similar to npm install) and then npm run build on every PR. (Also runs on any pushes to master.)

Close #57

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        node-version: [10.x, 12.x, 14.x, 15.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm ci
+    - run: npm run build --if-present
```
Comment (jack-lin-jql) [inline .github/workflows/build.yml:25]: Looks great! can you also add in `npm run lint` here?

**Comment by jack-lin-jql** [comment]
From my understanding, this build just builds with npm ci and build right? It doesn't actually deploy anything? (cuz like we don't want to deploy test apps to the master instance LOL

**Comment by JamesCahyadi** [comment]
> From my understanding, this build just builds with npm ci and build right? It doesn't actually deploy anything? (cuz like we don't want to deploy test apps to the master instance LOL)

Yeah, it doesn't actually deploy anything. I believe main.yml is responsible for deploying.

**Review by jack-lin-jql — APPROVED** [review]
LGTM

**Pull request merged**